SmartTutor/ContentModel/ContentService.cs

```diff
@@ -63,13 +66,16 @@ public NodeProgress GetNodeContent(int knowledgeNodeId, int? traineeId)
 
         private NodeProgress CreateNodeForTrainee(int knowledgeNodeId, int? traineeId)
         {
-            //TODO: Load Trainee prefs
-            //TODO: Get recommender to build NodeProgress with LOs for Trainee
-            //TODO: Save started NodeProgress to repo
+            if (traineeId == null) return null; // TODO: throw an exception?
+            var trainee = _traineeRepository.GetTraineeById(traineeId.Value);
+            var knowledgeNode = _lectureRepository.GetKnowledgeNodeWithSummaries(knowledgeNodeId);
+
+            var nodeProgress = _recommender.BuildNodeProgressForTrainee(trainee, knowledgeNode);
+            _traineeRepository.SaveNodeProgress(nodeProgress);
```

Comment:
This scenario assumes the Trainee did not already start the Node. We should also consider when the Trainee has already started/finished a node - such a node would be loaded instead of constructed. We can address this scenario next, not in this PR.